app/helpers/api_helper.rb

```diff
@@ -0,0 +1,24 @@
+require 'pandoc-ruby'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.